examples/ml-slurm-v6.yaml

```diff
@@ -0,0 +1,267 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: ml-slurm-v6
+
+vars:
+  project_id:  ## Set project id here
+  deployment_name: ml-example-v6
+  region: asia-southeast1
+  zone: asia-southeast1-b
+  zones:
+  - asia-southeast1-a
+  - asia-southeast1-b
+  - asia-southeast1-c
+  new_image:
+    family: ml-slurm
+    project: $(vars.project_id)
+  disk_size_gb: 200
+
+# Recommended to use GCS backend for Terraform state
+# See https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/examples#optional-setting-up-a-remote-terraform-state
+#
+# terraform_backend_defaults:
+#  type: gcs
+#  configuration:
+#    bucket: <<BUCKET_NAME>>
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network
+    source: modules/network/vpc
+
+  - id: homefs
+    source: modules/file-system/filestore
+    use:
+    - network
+    settings:
+      local_mount: /home
+      size_gb: 2560
+      filestore_tier: BASIC_SSD
+
+  - id: script
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - type: data
+        destination: /etc/yum.repos.d/google-fast-socket.repo
+        content: |
+          [google-fast-socket]
+          name=Google Cloud SDK
+          baseurl=https://packages.cloud.google.com/yum/repos/google-fast-socket
+          enabled=1
+          gpgcheck=1
+          repo_gpgcheck=0
+          gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+                 https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+      - type: shell
+        destination: install-ml-libraries.sh
+        content: |
+          #!/bin/bash
+          # this script is designed to execute on Slurm images published by SchedMD that:
+          # - are based on Debian 11 distribution of Linux
+          # - have NVIDIA Drivers v530 pre-installed
+          # - have CUDA Toolkit 12.1 pre-installed.
+
+          #set -e -o pipefail
```

Comment:
I think you unintentionally commented `set -e -o pipefail` out.

Reply:
Thanks for pointing this out. Updated.